### ISSUE TITLE
pep8speaks not working

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/tszip#5]
I enabled the [pep8speaks](https://pep8speaks.com/) GitHub app for this repo, but it doesn't seem to to be working.

**Comment by OrkoHunter** [comment]
Hi @jeromekelleher !

Thank you for raising the issue. Upon investigation, I found everything as expected. So, I created https://github.com/tskit-dev/tszip/pull/6 to see if the bot comments on an obvious error, and it did.

I think you expected a comment on https://github.com/tskit-dev/tszip/pull/4 and did not find anything. Adding `no_blank_comment: False` is really the way to go (which you have already done in the PR). 

From https://github.com/OrkoHunter/pep8speaks#configuration

> For every Pull Request, the bot looks for `.pep8speaks.yml` in the base branch (the existing one). If the file is not found, it then searches the head branch (the incoming changes).

Since the master branch already has a pep8speaks configuration file, that will be used by the bot, which says not to comment if no errors are found.
